guacamole-docker/bin/start.sh

```diff
@@ -583,8 +583,11 @@ END
 start_guacamole() {
 
     # Install webapp
-    rm -Rf /usr/local/tomcat/webapps/${WEBAPP_CONTEXT:-guacamole}
-    ln -sf /opt/guacamole/guacamole.war /usr/local/tomcat/webapps/${WEBAPP_CONTEXT:-guacamole}.war
+    if [ -n "$WEBAPP_CONTEXT" ]
+    then
+        rm /usr/local/tomcat/webapps/guacamole.war
+        ln -sf /opt/guacamole/guacamole.war /usr/local/tomcat/webapps/${WEBAPP_CONTEXT}.war
+    fi
```

Comment:
I'm not sure I understand how this both resolves the non-root user issue and also maintains the `WEBAPP_CONTEXT` capability?  It seems like, if linking at start time rather than build time causes non-root startup to fail, then anyone who is using `WEBAPP_CONTEXT` rather than the default `guacamole.war` is going to still have an issue running under a non-root account, correct??

Reply:
We can't do both, at least I did not find a way to do so.
We could `chmod 777 webapps` directory, but sounds like a security breach to me...
IMO there's no way to support `WEBAPP_CONTEXT` as non-root user...

Reply:
Finally solved with my last commit 👍